src/Filter/Filter.php

```diff
@@ -248,6 +248,24 @@ public function getRenderSettings(): array
         ];
     }
 
+    final public function showFilter(): ?bool
+    {
+        return $this->getOption('show_filter');
+    }
+
+    /**
+     * @return array<string, mixed>
+     */
+    final public function getLabelTranslationParameters(): array
+    {
+        return $this->getOption('label_translation_parameters');
```

Comment:
This might break your application if `label_translation_parameters` is null

Reply:
Indeed, please take a look at https://github.com/sonata-project/SonataAdminBundle/pull/8175